crates/circuit/src/dag_circuit.rs

```diff
@@ -70,7 +70,8 @@ use std::f64::consts::PI;
 #[cfg(feature = "cache_pygates")]
 use std::sync::OnceLock;
 
-static CONTROL_FLOW_OP_NAMES: [&str; 4] = ["for_loop", "while_loop", "if_else", "switch_case"];
+pub(crate) static CONTROL_FLOW_OP_NAMES: [&str; 4] =
```

Comment:
With the above comment we don't need this change.

```suggestion
static CONTROL_FLOW_OP_NAMES: [&str; 4] =
```